015-kafka-topic-encryption.md

```diff
@@ -97,38 +93,37 @@ Envoy however is but one viable approach, but certainly not the only means, to e
 in a proxy. 
```

Comment:
"Envoy however is but one viable approach, but certainly not the only means, to embed topic encryption in a proxy." Is that a correct English sentence? Sounds strange in my non-native ears.

Reply:
the English is correct. Stylistically there should be one "but". I will update.

Reply:
Just pushed a revised wording

Reply:
Hm, to me it still sounds strange. I would have removed the second "but". But I'm not the one who can judge this :)